tests/stencila-py/pyjp/verify

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+jupyter serverextension list 2>&1 | grep nbstencilaproxy
+jupyter nbextension list 2>&1 | grep nbstencilaproxy
+python3 -c "import stencila" 2>&1 | grep ModuleNotFoundError
```

Comment:
does the `pyjp` test check for something that the `py` test doesn't already cover? I was expecting to somehow run directly from a notebook or some such but it looks like this uses the manifest file and the jats file like the other test?

Reply:
The difference is in the used execution contexts that are installed. If the document only has `pyjp` cells, then the module `stencila/py` is not needed, because only Jupyter will be used to execute cells. 

In a previous version if handled the `pyjp XOR py` case in the same way, just installed the package. This is now more specific, but an extra case to handle/tests. _Should I remove this?_

Reply:
The difference is in the used execution contexts that are installed. If the document only has `pyjp` cells, then the module `stencila/py` is not needed, because only Jupyter will be used to execute cells. 

In a previous version if handled the `pyjp XOR py` case in the same way, just installed the package. This is now more specific, but an extra case to handle/test.

Reply:
The difference is in the used execution contexts that are installed. If the document only has `pyjp` cells, then the module `stencila/py` is not needed, because only Jupyter will be used to execute cells. 

In a previous version if handled the `pyjp XOR py` case in the same way, just installed the package. This is now more specific, but an extra case to handle/test.

Reply:
Ah ok, I didn't know that there were two ways of executing Python code. Thanks!